pvanalytics/quality/irradiance.py

```diff
@@ -289,7 +314,7 @@ def check_irradiance_consistency_qcrad(solar_zenith, ghi, dhi, dni,
                                        param=None, outside_domain=False):
     r"""Check consistency of GHI, DHI and DNI using QCRad criteria.
 
-    Uses criteria given in [1]_ to validate the ratio of irradiance
+    Uses criteria given in [1]_, [2]_ to validate the ratio of irradiance
```

Comment:
Are the consistency criteria the same in both papers?

Reply:
[1] only gives one set, but [2] gives both sets. So one might be inclined to only reference [2]. However, [2] is merely a two-page document whereas [1] is a journal article. I have been in correspondence with Christian Gueymard, who recommended referring to both sets of coefficients as QCRad tests as they were developed in this framework.

Reply:
For the *consistency* function, I don't see any point in referencing [2].  Seems like it doesn't offer anything extra over the journal article.